packages/chrome/src/elements/subnav/SubNavItem.tsx

```diff
@@ -11,6 +11,7 @@ import { StyledSubNavItem } from '../../styled';
 import { useChromeContext } from '../../utils/useChromeContext';
 
 interface ISubNavItemProps {
+  /** Adds the current item to the subnav */
   isCurrent?: boolean;
```

Comment:
This prop does not add an item to the subnav. It indicates which item in the subnav is current.

Reply:
Updated in latest push.